### DISCUSSION TITLE
New Unraid CA Docker error and warning messages

### DISCUSSION THREAD

**Discussion by motif82** [arabcoders/watchstate#348] [General] [answered]
Hi, I am trying to get watchstate set up to sync between Plex, Emby, and Jellyfin on my unraid server using the community application docker. Making progress and I think I have most things figured out but when I check the log after starting, I am getting a warning and error message and I'm not sure what I need to do to fix them or if I can just ignore them. 

[2023-10-06T16:21:47-0500] INFO: No environment file present at [/config/config/.env].
[2023-10-06T16:21:47-0500] Starting Cache Server.
**12:C 06 Oct 2023 16:21:47.861 # WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.**
[2023-10-06T16:21:47-0500] Starting HTTP Server.
{"level":"info","ts":1696627307.9501042,"msg":"using provided configuration","config_file":"/opt/config/Caddyfile","config_adapter":""}
{"level":"info","ts":1696627307.951969,"logger":"admin","msg":"admin endpoint started","address":"localhost:2019","enforce_origin":false,"origins":["//127.0.0.1:2019","//localhost:2019","//[::1]:2019"]}
{"level":"warn","ts":1696627307.9520895,"logger":"http.auto_https","msg":"server is listening only on the HTTP port, so no automatic HTTPS will be applied to this server","server_name":"srv0","http_port":8080}
{"level":"info","ts":1696627307.952095,"logger":"tls.cache.maintenance","msg":"started background certificate maintenance","cache":"0xc00067e700"}
{"level":"info","ts":1696627307.952336,"logger":"http.log","msg":"server running","name":"srv0","protocols":["h1","h2","h3"]}
**{"level":"error","ts":1696627307.952377,"msg":"unable to create folder for config autosave","dir":"/.config/caddy","error":"mkdir /.config: permission denied"}**
{"level":"info","ts":1696627307.952392,"msg":"serving initial configuration"}
{"level":"info","ts":1696627307.9523952,"logger":"tls","msg":"cleaning storage unit","description":"FileStorage:/.local/share/caddy"}
{"level":"info","ts":1696627307.9524212,"logger":"tls","msg":"finished cleaning storage units"}
Successfully started Caddy (pid=35) - Caddy is running in the background
[2023-10-06T16:21:47-0500] Caching tool Routes.
[2023-10-06T16:21:48-0500] Running database migrations.
[2023-10-06T16:21:48-0500] Running database maintenance tasks.
[2023-10-06T16:21:51-0500] Ensuring State table has correct indexes.
[2023-10-06T16:21:51-0500] Starting Tasks Scheduler.
[2023-10-06T16:21:51-0500] Running - WatchState 20230915-063bdf4
[06-Oct-2023 16:21:51] NOTICE: fpm is running, pid 1
[06-Oct-2023 16:21:51] NOTICE: ready to handle connections

Any help would be appreciated!

**Comment by arabcoders** [comment, marked as answer]
Do not worry about those two errors they aren't tool related and can be safely ignored. 

The first error is about generating config directory for compiled caddyfile and can be safely ignored.

The second error is related to redis key store which we use to store cache related data. it can also be ignored.

I will see if i can hide these errors  in future push

Thanks for reminding me.

**Reply by motif82** [reply]
Thanks for the quick reply!
